cypress/page/shoping-cart.page.ts

```diff
@@ -4,8 +4,8 @@ class ShopingCartPage {
     private goToCheckoutBtn: string;
 
     constructor() {
-        this.goToCartBtn = "[style*='display: block;'] .button-container > a";
-        this.goToCheckoutBtn = ".cart_navigation span";
+        this.goToCartBtn = ".button-container > .button-medium";
+        this.goToCheckoutBtn = ".cart_navigation > .standard-checkout";
```

Comment:
can you use an attribute to indicate we are referring to the proceed to checkout button? you can check if the contains works here